roles/common/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+# All containers need to be up in order for this task to pass
+- name: "[TEST] verify status of multiple containers"
+  ansible.builtin.include_tasks:
+    file: container_status.yml
+  loop: "{{ containers }}"
+  loop_control:
+    label: "{{ item }}"
+  vars:
+    container_name: "{{ item }}"
+
```

Comment:
This is separate from adding the service checks. 

It should NOT be run unconditionally